editor/editor_help.cpp

```diff
@@ -1849,7 +1849,7 @@ void EditorHelp::_update_doc() {
 				_push_code_font();
 
 				if (constant.value.begins_with("Color(") && constant.value.ends_with(")")) {
-					String stripped = constant.value.replace(" ", "").replace("Color(", "").replace(")", "");
+					String stripped = constant.value.remove_char(' ').replace("Color(", "").remove_char(')');
```

Comment:
Could change this to:
```suggestion
					String stripped = constant.value.remove_chars({ ' ', ')' }).replace("Color(", "");
```
But since the original code keeps the two removes separate I'm unsure if it might cause some issues so left it for now, but if it's considered safe I'll change this